device.go

```diff
@@ -12,6 +12,18 @@ import (
 	"time"
 )
 
+const (
```

Comment:
Unexport all of these and change to camel case `brightnessMin` etc and just mention the allowed limits on the field docs within Light.